Sources/Swiftly/ListAvailable.swift

```diff
@@ -45,18 +45,22 @@ struct ListAvailable: SwiftlyCommand {
             try ToolchainSelector(parsing: input)
         }
 
-        if let selector {
-            guard !selector.isSnapshotSelector() else {
-                SwiftlyCore.print("Listing available snapshots is not currently supported.")
-                return
+        let config = try Config.load()
+
+        let tc: [ToolchainVersion]
+
+        switch selector {
+        case let .snapshot(branch, _):
+            if case let .release(major, _) = branch, major < 6 {
+                throw Error(message: "Listing available snapshots previous to 6.0 is not supported.")
```

Comment:
If we're throwing an error here, we can remove the GitHub fallback.

Reply:
The GitHub fallback is still there in case someone wants to directly `swiftly install` a snapshot from pre-6.0 branch. They just can't list them using this subcommand.